android/modules/media/src/java/ti/modules/titanium/media/TiCameraActivity.java

```diff
@@ -161,6 +164,38 @@ public void onCreate(Bundle savedInstanceState)
 
 	}
 
+	// This is overriden to prevent using the FullScreen method which is not supported for Camera use.
+	@Override
+	protected void windowCreated(Bundle savedInstanceState) {
```

Comment:
This override may not be required. 

TiCameraActivity is a defined activity in manifest file which is used only for camera preview. So fullscreen is always `false` here
https://github.com/ashcoding/titanium_mobile/blob/TIMOB-19769-B/android/titanium/src/java/org/appcelerator/titanium/TiBaseActivity.java#L500

So override is unnecessary.